### PR TITLE
Add update notifications

### PR DESCRIPTION
**Pull request by vadi2**
Peek 2020-04-03 15-51

Update check happens 2s after launch and it shows a non-intrusive link to get the latest release. The admin is texted of the latest version - once per new release - so if they skip a release, they'll still get a notification for the next one.

Development builds don't include update checks by default so it's not annoying.

### CODE REVIEW

**Comment by grahamegrieve** [comment]
@brianpos Vadim is using System.Diagnostics.Trace.WriteLine()... is that a problem? We want to use NLog consistently?

**Comment by vadi2** [comment]
This isn't ready yet!

Trace - I found an example of that in the code - happy to change to NLog.

**Comment by vadi2** [comment]
This is now ready for review.

File: ClinicArrivals/ViewModel.cs [hunk 1]
```diff
@@ -69,6 +85,13 @@ public ViewModel()
                 FhirApptUpdater = new FhirAppointmentUpdater(FhirAppointmentReader.GetServerConnection);
                 SmsProcessor = new TwilioSmsProcessor();
             }
+
+            // ensure default is 1.2+ as Github dropped support for 1.1
+            ServicePointManager.SecurityProtocol |= SecurityProtocolType.Tls12;
```
Comment (jamesianberry) [inline ClinicArrivals/ViewModel.cs:90]: So this enables TLS1.2 if not already enabled but does not turn off earlier protocols, right?
Side note: this seems to be the MS guidance for TLS in our situation https://docs.microsoft.com/en-us/dotnet/framework/network-programming/tls#if-your-app-targets-a-net-framework-version-earlier-than-47
At the top of the article the guidance is:
- Do not specify the TLS version. Configure your code to let the OS decide on the TLS version.
- Perform a thorough code audit to verify you're not specifying a TLS or SSL version.

However in our situation we know that TLS1.2 is the best that .NET 4.62 will support, and in TwilioSmsProcessor.cs we have (2x): ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12;

**Review by jamesianberry — APPROVED** [review]

File: ClinicArrivals/ViewModel.cs (hunk 1 above)
Comment (vadi2) [reply ClinicArrivals/ViewModel.cs:90]: I think so. In my Win10 Pro windows VM it seemed to work okay - if there's any issues with earlier Windows installs, happy to fix.

**Pull request merged**